packages/babel-plugin-transform-es2015-template-literals/README.md

```diff
@@ -75,7 +75,7 @@ In loose mode, tagged template literal objects aren't frozen.
 
 `boolean`, defaults to `false`.
 
-This option wraps all template literal expressions with `String`. See [babel/babel#1065](https://github.com/babel/babel/issues/1065) for more info.
```

Comment:
@hzoo I think we can add a link to this PR in README. Also going to add a few sentences about spec option and why we are using `String.prototype.concat`.